adele-order-manager/src/main/java/com/microservicesteam/adele/ordermanager/domain/OrderRepository.java

```diff
@@ -9,20 +9,20 @@
 @Repository
 public interface OrderRepository extends JpaRepository<Order, String> {
 
-    @Modifying(clearAutomatically = true)
-    @Query("update Orders o set o.paymentId = :paymentId where orderId = :orderId")
-    int updatePaymentId(@Param("orderId") String orderId, @Param("paymentId") String paymentId);
+    @Modifying
+    @Query("update Order o set o.paymentId = :paymentId where o.orderId = :orderId")
```

Comment:
Shouldn't we refer to table name 'Orders'?

Reply:
no